easycv/predictors/detector.py

```diff
@@ -38,6 +38,190 @@
     from easycv.thirdparty.mtcnn import FaceDetector
 
 
+@PREDICTORS.register_module()
+class DetectorPredictor(PredictorInterface):
```

Comment:
it does not support all the detectors, rename it  for this time and give it back when it support all the detection moduels

Reply:
done